### PR TITLE
Add docker development environment

### PR DESCRIPTION
**Pull request by subnomo**
Add docker development environment. The instructions to get started are in the README.

### CODE REVIEW

File: run.py [hunk 1]
```diff
@@ -1,3 +1,3 @@
 from app import app
 
-app.run()
+app.run(host='0.0.0.0')
```
Comment (pop) [inline run.py]: make this a config file variable. Also make the `port` a config file variable.

File: dockerfiles/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,9 @@
+FROM centos:latest
+RUN yum -y update
+RUN yum -y install epel-release
+RUN yum -y groupinstall "Development Tools"
+RUN yum -y install python-pip python-devel libffi-devel
+
+COPY requirements.txt /opt/
+RUN pip install -r /opt/requirements.txt
+EXPOSE 5000
```
Comment (pop) [inline dockerfiles/Dockerfile]: I think this is actually extraneous. Try getting rid of it.

**Comment by pop** [comment]
lgtm +1

**Comment by Kennric** [comment]
Looks good. +1

**Pull request merged**